cmd/snap/cmd_connect_test.go

```diff
@@ -30,27 +30,19 @@ import (
 
 func (s *SnapSuite) TestConnectHelp(c *C) {
 	msg := `Usage:
-  snap.test [OPTIONS] connect <snap>:<plug> <snap>:<slot>
+  snap.test [OPTIONS] connect [<snap>:<plug>] [<snap>:<slot>]
 
 The connect command connects a plug to a slot.
 It may be called in the following ways:
 
 $ snap connect <snap>:<plug> <snap>:<slot>
 
-Connects the specific plug to the specific slot.
+Connects the provided plug to the given slot.
 
-$ snap connect <snap>:<plug> <snap>
+$ snap connect <snap>:<plug>
 
-Connects the specific plug to the only slot in the provided snap that matches
-the connected interface. If more than one potential slot exists, the command
-fails.
```

Comment:
This is the additional case we discussed, and it looks like it is now supported?  Can you please copy over this docs to the new help?